internal/worker/remoterelationconsumer/applicationworker.go

```diff
@@ -97,17 +98,16 @@ func (c RemoteApplicationConfig) Validate() error {
 	return nil
 }
 
-// remoteApplicationWorker listens for localChanges to relations
-// involving a remote application, and publishes change to
-// local relation units to the remote model. It also watches for
-// changes originating from the offering model and consumes those
-// in the local model.
+// remoteApplicationWorker listens for consumer changes to relations involving a
```

Comment:
"involving an offered application"

"publishes ... to the offering model"

I would also call this `localConsumerWorker`.